common/supplemental/supplementalMetadata.xml

```diff
@@ -1572,11 +1585,8 @@ For terms of use, see http://www.unicode.org/copyright.html
 						
 			
             <!-- variants -->
-            <variantAlias type="AALAND" replacement="AX" reason="deprecated"/>
-            <variantAlias type="POLYTONI" replacement="POLYTON" reason="deprecated"/>
-            <variantAlias type="HEPLOC" replacement="ALALC97" reason="deprecated"/>
-			<variantAlias type="arevela" replacement="hy" reason="deprecated"/> <!-- arevela -->
-			<variantAlias type="arevmda" replacement="hyw" reason="deprecated"/> <!-- arevmda -->
+            <variantAlias type="aaland" replacement="AX" reason="deprecated"/>
+            <variantAlias type="polytoni" replacement="polyton" reason="deprecated"/>
```

Comment:
if the type in variantAlias is containing not just variant code, but any of l,s,r,v combination, then these should be
<variantAlias type="aaland" replacement="und-AX" reason="deprecated"/>
<variantAlias type="polytoni" replacement="und-polyton" reason="deprecated"/>
<variantAlias type="heploc" replacement="und-alalc97" reason="deprecated"/>

Reply:
Maybe the best approach is that languageAlias has language *tags* as both type and replacement. Others must never change type. That is, instead of 
<variantAlias type="aaland" replacement="und-AX" reason="deprecated"/>
we should have:
<languageAlias type="und_aaland" replacement="und_AX" reason="deprecated"/>